.github/workflows/ci.yml

```diff
@@ -27,6 +27,27 @@ jobs:
             ${{ runner.os }}-turbo-
       - run: yarn turbo run --filter='./packages/node*' lint
       - run: yarn turbo run --filter='./packages/node*' test
+  analytics-node-cf-workers:
+    name: 'analytics-node QA (Cloudflare Workers)'
```

Comment:
Might it make sense to just fold this into the consent-intg-tests job and run everything on node 18? Naming is a bit weird now with a second category of  integration tests